lib/block_view_bitcoin_test.go

```diff
@@ -57,10 +58,7 @@ func GetTestParamsCopy(
 		// blocks that follow it, it hurdles the min chain work.
 		big.NewInt(0),
 		// We are bastardizing the DeSo header to store Bitcoin information here.
-		&MsgDeSoHeader{
-			TstampSecs: uint64(startHeader.Timestamp.Unix()),
-			Height:     0,
-		},
+		(&MsgDeSoHeader{Height: 0}).SetTstampSecs(uint64(startHeader.Timestamp.Unix())),
```

Comment:
I don't love the way this looks. We could just set `TstampNanoSecs` on the struct, but I see why we did it this way to make the change less significant.